editor/import/resource_importer_scene.cpp

```diff
@@ -372,6 +372,194 @@ static void _pre_gen_shape_list(Ref<ImporterMesh> &mesh, Vector<Ref<Shape3D>> &r
 	}
 }
 
+struct ScalableNodeCollection {
+	List<Node3D *> node_3ds;
```

Comment:
Good catch that heap List is not appropriate.

But I actually believe these should be Set<> because it is doing .find() followed by .push_back() which will cause an O(N^2) order complexity.

Set is fine because processing data can happen in arbitrary order